0001_education_matters_addChild.js

```diff
@@ -0,0 +1,72 @@
+const Common = require("../../../../setup/common.js");
+
+const folderName = __dirname.match(/[^\\/]+$/);
+Cypress.on("uncaught:exception", () => false);
+Cypress.on("uncaught:exception", (err) => {
+   if (
+      err.message.includes(
+         "TypeError: Cannot read properties of undefined (reading 'raw')"
+      )
+   ) {
+      return false;
+   }
+});
+
+before(() => {
+   Common.ResetDB(cy);
+   Common.AuthLogin(cy);
+   cy.request("POST", "/test/import", {
+      file: `imports/${folderName}/app_Education_Matters_20221003.json`,
+   });
+});
+
+beforeEach(() => {
+   Common.AuthLogin(cy);
+   Common.RunSQL(cy, folderName, ["reset_tables.sql", "insert_data.sql"]);
+   // Open the App
+   cy.visit("/");
+   cy.get('[data-cy="portal_work_menu_sidebar"]').should("exist").click();
+   cy.get("[data-cy='ab7ab867-549a-498f-a1ff-a9e2f97dd525']")
+      .should("exist")
+      .click();
+});
+
+describe("Adding Child", () => {
+   it.only("Can Add New Child", () => {
```

Comment:
Since we only have one test, its not a big deal. But should remove `only` so it easier to add tests in the future.

Reply:
It's solved